projects/valtimo/plugin/src/lib/plugins/open-zaak/components/set-resultaat-configuration/set-resultaat-configuration.component.ts

```diff
@@ -28,4 +31,68 @@ export class SetResultaatConfigurationComponent implements PluginConfigurationCo
   @Output() valid: EventEmitter<boolean> = new EventEmitter<boolean>();
   @Output() configuration: EventEmitter<PluginConfigurationData> =
     new EventEmitter<PluginConfigurationData>();
+
+  private readonly selectedResultaat$ = new BehaviorSubject<string>('');
+
+  readonly selectedZaakType$ = new BehaviorSubject<ZaakType | null>(null);
+
+  readonly RESULTATEN: Array<string> = [
+    'Ingetrokken',
+    'Ongegrond met financ',
```

Comment:
Is this spelling correct? Looks like you ment 'financië' here, the same as below?